src/libcollections/slice.rs

```diff
@@ -343,15 +343,17 @@ impl<T> [T] {
         core_slice::SliceExt::last_mut(self)
     }
 
-    /// Returns the element of a slice at the given index, or `None` if the
-    /// index is out of bounds.
+    /// Returns the element or subslice of a slice at the given position or
```

Comment:
Tried to do that now without too much repetition. Let me know if you want the details in all the method docs.